app/version.txt

```diff
@@ -1 +1 @@
-Build:  18-Jul-23
\ No newline at end of file
+Build:  lldpd
```

Comment:
Why did this change to the date instead of the version? Now its the branch name? I am confused.
I preferred seeing the actual version which is much easer to maintain.
```suggestion
Build:  v1.2.0
```